src/app/(proper_react)/(redesign)/(public)/how-it-works/components/DataBrokers.tsx

```diff
@@ -0,0 +1,163 @@
+              {l10n.getString("section-1-step-1-subtitle")}
+            </h3>
+            <div className={styles.privateInfoRow}>
+              <Image src={NameIcon} alt="" />
+              <p className={styles.privateInfoText}>
+                {l10n.getFragment("section-1-step-1-name", {
+                  elems: {
+                    name: <span className={styles.boldedText} />,
+                  },
+                })}
+              </p>
+            </div>
+            <div className={styles.privateInfoRow}>
+              <Image src={LocationIcon} alt="" />
+              <p className={styles.privateInfoText}>
+                {l10n.getFragment("section-1-step-1-location", {
+                  elems: {
+                    location: <span className={styles.boldedText} />,
+                  },
+                })}
+              </p>
+            </div>
+            <div className={styles.privateInfoRow}>
+              <Image src={DOBIcon} alt="" />
+              <p className={styles.privateInfoText}>
+                {l10n.getFragment("section-1-step-1-dob", {
+                  elems: {
+                    dob: <span className={styles.boldedText} />,
+                  },
+                })}
+              </p>
+            </div>
+            <p className={styles.stepBody}>
+              {l10n.getString("section-1-step-1-closing-text")}
+            </p>
+          </div>
+          <Image src={RemoveStep1} alt="step 1 screenshot" />
+        </li>
+        {/* STEP 2 */}
+        <li className={`${styles.step} ${styles.reversedStep}`}>
+          <Image src={RemoveStep2} alt="" />
+          <div className={styles.stepTextContainer}>
+            <h3 className={`${styles.brokersEmphasis} ${styles.stepTitle}`}>
+              {l10n.getString("section-1-step-2-title")}
+            </h3>
+            <h3 className={styles.stepSubtitle}>
+              {l10n.getString("section-1-step-2-subtitle")}
+            </h3>
+            <p className={styles.stepBody}>
+              {l10n.getString("section-1-step-2-text-1")}
+            </p>
+            <p className={styles.stepBody}>
+              {l10n.getString("section-1-step-2-text-2")}
+            </p>
+          </div>
+        </li>
+        {/* STEP 3 */}
+        <li className={styles.step}>
+          <div className={styles.stepTextContainer}>
+            <h3 className={`${styles.brokersEmphasis} ${styles.stepTitle}`}>
+              {l10n.getString("section-1-step-3-title")}
+            </h3>
+            <h3 className={styles.stepSubtitle}>
+              {l10n.getString("section-1-step-3-subtitle")}
+            </h3>
+            <p className={styles.stepBody}>
+              {l10n.getString("section-1-step-3-text-1")}
+            </p>
+            <p className={styles.stepBody}>
+              {l10n.getString("section-1-step-3-text-2")}
+            </p>
+          </div>
+          <Image src={RemoveStep3} alt="" />
+        </li>
```

Comment:
I like this suggestion, but the DataBreaches and DataBrokers have different styles in terms of background color and latter has an image for each step while the former has one image for three steps.  I think it is doable, but am concerned that the time spent on this could delay launch.  Would this be better as a tech debt item?